WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -516,13 +516,25 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
                 guard isEditable else {
                     return
                 }
-                // TODO: 11090 - Analytics
+
+                if product is EditableProductVariationModel {
+                    ServiceLocator.analytics.track(.productVariationViewSubscriptionFreeTrialTapped)
+                } else if product is EditableProductModel {
+                    ServiceLocator.analytics.track(.productDetailsViewSubscriptionFreeTrialTapped)
+                }
+
                 showSubscriptionFreeTrialSettings()
             case .subscriptionExpiry(_, let isEditable):
                 guard isEditable else {
                     return
                 }
-                // TODO: 11090 - Analytics
+
+                if product is EditableProductVariationModel {
+                    ServiceLocator.analytics.track(.productVariationViewSubscriptionExpirationDateTapped)
+                } else if product is EditableProductModel {
+                    ServiceLocator.analytics.track(.productDetailsViewSubscriptionExpirationDateTapped)
+                }
```

Comment:
Same as the other comment. We should use the `eventLogger` to log these events.

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/11236/commits/ad05c6db36ae416e9b1e941423d8dace72e31e10